spec/index.bs

```diff
@@ -114,33 +168,131 @@ interface GPUAdapter : GPUObjectBase {
 };
 </script>
 
+{{GPUAdapter}} has the following attributes:
+
+<dl dfn-type=attribute dfn-for=GPUAdapter>
+    : <dfn>\[[adapter]]</dfn>, of type [=adapter=], readonly
+    ::
+        An internal slot holding the [=adapter=] to which this {{GPUAdapter}}
+        refers.
+
+    : <dfn>name</dfn>
+    ::
+        A human-readable name identifying the adapter.
+        The contents are implementation-defined.
+
+    : <dfn>extensions</dfn>
+    ::
+        A {{GPUExtensions}} object which enumerates the extensions supported
+        by the user agent, and whether each extension is supported by the
```

Comment:
It's strange to see "user agent" here. Why aren't we saying "this adapter"?

Reply:
Because of the logic just below this (which distinguishes UA support from hardware support). Do you think we should not distinguish the two? - `undefined` or `true` (I'm ok with this.)

Reply:
We should probably discuss that on a call. It's not obvious to me what benefit there is to specify UA and hardware as two different things as opposed to a single "UA's view of the hardware"

Reply:
Sounds like both changes (false->undefined and UA+hardware merge) would remove useless complexity, which is good by me.